From 1494d8daacfbfa623447b7937ce50927c01c832d Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Sat, 26 Sep 2026 15:14:42 +0000
Subject: [PATCH] Refresh the guest package lock

alsa-card-profiles 1:1.6.8-1 -> 1:1.6.9-1
aom 3.15.0-1 -> 3.15.1-1
bash 5.3.15-1 -> 5.3.20-1
bat 0.26.1-2 -> 0.26.1-3
bubblewrap 0.12.0-1 -> 0.13.0-1
ca-certificates-mozilla 3.129-1 -> 3.130-1
cairo 1.18.4-1 -> 1.18.6-1
expat 2.8.4-1 -> 2.8.5-1
fcitx5 5.1.22-1 -> 5.1.23-1
fcitx5-qt 5.1.15-1 -> 5.1.16-1
ffmpeg 2:9.0.1-4 -> 2:9.0.2-1
fribidi 1.0.16-2 -> 1.0.17-1
gpu-screen-recorder 6.1.2-1 -> 6.1.3-1
gst-plugins-bad-libs 1.28.7-1 -> 1.28.7-2
gst-plugins-base-libs 1.28.7-1 -> 1.28.7-2
gstreamer 1.28.7-1 -> 1.28.7-2
gum 2.0.1-1 -> 2.0.2-1
gzip 1.14-2 -> 1.15-1
harfbuzz 14.4.0-1 -> 14.5.0-1
hyprland-guiutils 0.2.2-3 -> 0.2.2-4
hyprtoolkit 0.5.4-6 -> 0.6.0-1
imagemagick 7.1.2.31-1 -> 7.1.2.31-2
less 1:704-1 -> 1:710-1
libdwarf 1:2.3.2-1 -> 1:2.3.3-1
libheif 1.23.4-1 -> 1.23.5-1
libpipewire 1:1.6.8-1 -> 1:1.6.9-1
libvips 8.18.6-1 -> 8.18.6-2
libwireplumber 0.5.17-1 -> 0.5.17-2
linux 7.2.6.arch2-1 -> 7.2.7.arch1-1
linux-headers 7.2.6.arch2-1 -> 7.2.7.arch1-1
mujs 1.3.9-1 -> 1.3.10-1
nss 3.129-1 -> 3.130-1
openexr 3.4.15-1 -> 3.5.1-1
openjph 0.31.0-1 -> 0.32.0-1
parted 3.7-1 -> 3.8-1
pcsclite 2.5.1-1 -> 2.5.2-1
pipewire 1:1.6.8-1 -> 1:1.6.9-1
pipewire-alsa 1:1.6.8-1 -> 1:1.6.9-1
pipewire-audio 1:1.6.8-1 -> 1:1.6.9-1
pipewire-pulse 1:1.6.8-1 -> 1:1.6.9-1
readline 8.3.003-1 -> 8.3.6-1
systemd 261.3-1 -> 262-1
systemd-libs 261.3-1 -> 262-1
systemd-resolvconf 261.3-1 -> 262-1
systemd-sysvcompat 261.3-1 -> 262-1
util-linux 2.42.3-1 -> 2.42.4-1
util-linux-libs 2.42.3-1 -> 2.42.4-1
wireplumber 0.5.17-1 -> 0.5.17-2
---
 guest/packages.lock.json | 96 ++++++++++++++++++++--------------------
 1 file changed, 48 insertions(+), 48 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index a485361..074fa5c 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -7,11 +7,11 @@
     "adwaita-fonts": "51.0-2",
     "adwaita-icon-theme": "50.0-1",
     "adwaita-icon-theme-legacy": "46.2-3",
-    "alsa-card-profiles": "1:1.6.8-1",
+    "alsa-card-profiles": "1:1.6.9-1",
     "alsa-lib": "1.2.16.1-1",
     "alsa-topology-conf": "1.2.5.1-4",
     "alsa-ucm-conf": "1.2.16.1-1",
-    "aom": "3.15.0-1",
+    "aom": "3.15.1-1",
     "appstream": "1.2.0-1",
     "aquamarine": "0.15.1-1",
     "archlinux-keyring": "20260909-1",
@@ -23,21 +23,21 @@
     "avahi": "1:0.9rc5-1",
     "base": "3-3",
     "base-devel": "1-2",
-    "bash": "5.3.15-1",
+    "bash": "5.3.20-1",
     "bash-completion": "2.18.0-1",
-    "bat": "0.26.1-2",
+    "bat": "0.26.1-3",
     "binutils": "2.47-4",
     "bison": "3.8.2-8",
     "bluez-libs": "5.87-2",
     "boost-libs": "1.92.0-1",
     "brotli": "1.2.0-1",
     "btop": "1.4.7-1",
-    "bubblewrap": "0.12.0-1",
+    "bubblewrap": "0.13.0-1",
     "bzip2": "1.0.8-6",
     "ca-certificates": "20240618-1",
-    "ca-certificates-mozilla": "3.129-1",
+    "ca-certificates-mozilla": "3.130-1",
     "ca-certificates-utils": "20240618-1",
-    "cairo": "1.18.4-1",
+    "cairo": "1.18.6-1",
     "cfitsio": "1:4.7.0-1",
     "chromium": "153.0.8010.52-1",
     "clang": "22.1.8-1",
@@ -68,16 +68,16 @@
     "enchant": "2.8.21-1",
     "exempi": "2.6.6-3",
     "exiv2": "0.28.9-1",
-    "expat": "2.8.4-1",
+    "expat": "2.8.5-1",
     "eza": "0.23.5-2",
     "fakeroot": "1:1.37.2-3",
     "fastfetch": "2.68.1-1",
     "fcft": "3.3.3-1",
-    "fcitx5": "5.1.22-1",
+    "fcitx5": "5.1.23-1",
     "fcitx5-gtk": "5.1.7-1",
-    "fcitx5-qt": "5.1.15-1",
+    "fcitx5-qt": "5.1.16-1",
     "fd": "10.5.0-3",
-    "ffmpeg": "2:9.0.1-4",
+    "ffmpeg": "2:9.0.2-1",
     "fftw": "3.3.11-1",
     "file": "5.48-1",
     "filesystem": "2025.10.12-1",
@@ -87,7 +87,7 @@
     "fontconfig": "2:2.18.3-2",
     "foot": "1.28.0-2",
     "freetype2": "2.14.3-1",
-    "fribidi": "1.0.16-2",
+    "fribidi": "1.0.17-1",
     "fuse-common": "3.18.3-1",
     "fuse3": "3.18.3-1",
     "fzf": "0.74.4-1",
@@ -126,7 +126,7 @@
     "gpgme": "2.2.0-1",
     "gpgmepp": "2.2.0-1",
     "gpm": "1.20.7.r38.ge82d1a6-7",
-    "gpu-screen-recorder": "6.1.2-1",
+    "gpu-screen-recorder": "6.1.3-1",
     "graphene": "1.10.8-2",
     "graphite": "1:1.3.15-1",
     "grep": "3.12-2",
@@ -136,22 +136,22 @@
     "gsettings-system-schemas": "50.1-1",
     "gsm": "1.0.24-1",
     "gssdp": "1.6.6-1",
-    "gst-plugins-bad-libs": "1.28.7-1",
-    "gst-plugins-base-libs": "1.28.7-1",
-    "gstreamer": "1.28.7-1",
+    "gst-plugins-bad-libs": "1.28.7-2",
+    "gst-plugins-base-libs": "1.28.7-2",
+    "gstreamer": "1.28.7-2",
     "gtest": "1.18.0-1",
     "gtk-update-icon-cache": "1:4.22.5-1",
     "gtk3": "1:3.24.52-1",
     "gtk4": "1:4.22.5-1",
     "gtk4-layer-shell": "1.3.0-1",
     "guile": "3.0.11-1",
-    "gum": "2.0.1-1",
+    "gum": "2.0.2-1",
     "gupnp": "1:1.6.10-1",
     "gupnp-dlna": "0.12.0-5",
     "gupnp-igd": "1.6.0-2",
     "gvfs": "1.60.3-3",
-    "gzip": "1.14-2",
-    "harfbuzz": "14.4.0-1",
+    "gzip": "1.15-1",
+    "harfbuzz": "14.5.0-1",
     "hdf5": "2.2.0-1",
     "herdr": "0.8.2-1",
     "hicolor-icon-theme": "0.18-1",
@@ -163,17 +163,17 @@
     "hyprgraphics": "0.5.1-4",
     "hypridle": "0.1.8-2",
     "hyprland": "0.56.2-3",
-    "hyprland-guiutils": "0.2.2-3",
+    "hyprland-guiutils": "0.2.2-4",
     "hyprland-preview-share-picker": "0.2.1-1",
     "hyprlang": "0.6.8-5",
     "hyprpicker": "0.4.7-4",
     "hyprsunset": "0.4.0-3",
-    "hyprtoolkit": "0.5.4-6",
+    "hyprtoolkit": "0.6.0-1",
     "hyprutils": "0.14.2-1",
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
-    "imagemagick": "7.1.2.31-1",
+    "imagemagick": "7.1.2.31-2",
     "imath": "3.2.3-1",
     "imv": "5.0.1-2",
     "iniparser": "4.3.0-1",
@@ -198,7 +198,7 @@
     "lcms2": "2.19.1-1",
     "leancrypto": "1.9.0-1",
     "leptonica": "1.87.0-2",
-    "less": "1:704-1",
+    "less": "1:710-1",
     "libadwaita": "1:1.9.4-1",
     "libaec": "1.1.7-1",
     "libarchive": "3.8.9-1",
@@ -244,7 +244,7 @@
     "libdrm": "2.4.134-1",
     "libdvdnav": "7.0.0-1",
     "libdvdread": "7.1.1-1",
-    "libdwarf": "1:2.3.2-1",
+    "libdwarf": "1:2.3.3-1",
     "libebur128": "1.2.6-2",
     "libedit": "20260512_3.1-1",
     "libei": "1.6.0-1",
@@ -269,7 +269,7 @@
     "libgsf": "1.14.58-1",
     "libgudev": "238-3",
     "libgxps": "0.3.2-5",
-    "libheif": "1.23.4-1",
+    "libheif": "1.23.5-1",
     "libhwasan": "16.2.1+r23+gd564253eb6c8-1",
     "libice": "1.1.2-1",
     "libidn2": "2.3.8-1",
@@ -324,7 +324,7 @@
     "libpciaccess": "0.19-1",
     "libpgm": "5.3.128-4",
     "libpipeline": "1.5.8-1",
-    "libpipewire": "1:1.6.8-1",
+    "libpipewire": "1:1.6.9-1",
     "libplacebo": "7.360.1-6",
     "libplist": "2.7.0-3",
     "libpng": "1.6.58-2",
@@ -375,14 +375,14 @@
     "libva": "2.24.1-1",
     "libvdpau": "1.5-4",
     "libverto": "0.3.2-6",
-    "libvips": "8.18.6-1",
+    "libvips": "8.18.6-2",
     "libvorbis": "1.3.7-4",
     "libvpl": "2.17.0-1",
     "libvpx": "1.17.0-1",
     "libvterm": "0.3.3-2",
     "libwacom": "2.20.0-1",
     "libwebp": "1.6.0-2",
-    "libwireplumber": "0.5.17-1",
+    "libwireplumber": "0.5.17-2",
     "libx11": "1.8.13-1",
     "libxau": "1.0.12-1",
     "libxcb": "1.17.0-1",
@@ -418,9 +418,9 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.6.arch2-1",
+    "linux": "7.2.7.arch1-1",
     "linux-api-headers": "7.2-1",
-    "linux-headers": "7.2.6.arch2-1",
+    "linux-headers": "7.2.7.arch1-1",
     "llhttp": "9.3.1-1",
     "llvm-libs": "22.1.8-2",
     "lm_sensors": "1:3.6.2-1",
@@ -449,7 +449,7 @@
     "mpv": "1:0.41.0-6",
     "msgpack-c": "7.0.1-1",
     "mtdev": "1.1.7-1",
-    "mujs": "1.3.9-1",
+    "mujs": "1.3.10-1",
     "muparser": "2.3.5-2",
     "nautilus": "50.3.1-1",
     "ncurses": "6.6-2",
@@ -462,7 +462,7 @@
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
     "nspr": "4.40-1",
-    "nss": "3.129-1",
+    "nss": "3.130-1",
     "numactl": "2.0.19-1",
     "ocl-icd": "2.3.5-1",
     "omacalc": "0.2.2-1",
@@ -474,10 +474,10 @@
     "oniguruma": "6.9.10-1",
     "openal": "1.25.2-2",
     "opencore-amr": "0.1.6-2",
-    "openexr": "3.4.15-1",
+    "openexr": "3.5.1-1",
     "openh264": "2.6.0-2",
     "openjpeg2": "2.5.4-1",
-    "openjph": "0.31.0-1",
+    "openjph": "0.32.0-1",
     "openssh": "10.5p1-1",
     "openssl": "3.6.4-1",
     "opus": "1.6.1-1",
@@ -492,21 +492,21 @@
     "pambase": "20260616-1",
     "pamixer": "1.6-4",
     "pango": "1:1.58.2-1",
-    "parted": "3.7-1",
+    "parted": "3.8-1",
     "patch": "2.8-1",
     "pciutils": "3.15.0-1",
     "pcre": "8.45-5",
     "pcre2": "10.48-1",
-    "pcsclite": "2.5.1-1",
+    "pcsclite": "2.5.2-1",
     "perl": "5.42.2-2",
     "perl-error": "0.17030-3",
     "perl-mailtools": "2.22-3",
     "perl-timedate": "2.35-1",
     "pinentry": "1.3.3-1",
-    "pipewire": "1:1.6.8-1",
-    "pipewire-alsa": "1:1.6.8-1",
-    "pipewire-audio": "1:1.6.8-1",
-    "pipewire-pulse": "1:1.6.8-1",
+    "pipewire": "1:1.6.9-1",
+    "pipewire-alsa": "1:1.6.9-1",
+    "pipewire-audio": "1:1.6.9-1",
+    "pipewire-pulse": "1:1.6.9-1",
     "pixman": "0.46.4-1",
     "pkgconf": "3.0.7-1",
     "playerctl": "2.4.1-5",
@@ -537,7 +537,7 @@
     "quickshell": "0.3.1-1",
     "rav1e": "0.8.1-3",
     "re2": "2:2025.11.05-6",
-    "readline": "8.3.003-1",
+    "readline": "8.3.6-1",
     "ripgrep": "15.2.0-1",
     "rubberband": "4.0.0-2",
     "sbc": "2.2-1",
@@ -568,10 +568,10 @@
     "starship": "1.26.0-1",
     "sudo": "1.9.17.p2-6",
     "svt-av1": "4.2.0-1",
-    "systemd": "261.3-1",
-    "systemd-libs": "261.3-1",
-    "systemd-resolvconf": "261.3-1",
-    "systemd-sysvcompat": "261.3-1",
+    "systemd": "262-1",
+    "systemd-libs": "262-1",
+    "systemd-resolvconf": "262-1",
+    "systemd-sysvcompat": "262-1",
     "tar": "1.35-5",
     "tensaku": "0.29.0-1",
     "tesseract": "5.5.3-1",
@@ -601,8 +601,8 @@
     "unibilium": "2.1.4-1",
     "unzip": "6.0-23",
     "upower": "1.91.4-1",
-    "util-linux": "2.42.3-1",
-    "util-linux-libs": "2.42.3-1",
+    "util-linux": "2.42.4-1",
+    "util-linux-libs": "2.42.4-1",
     "uwsm": "0.27.0-1",
     "v4l-utils": "1.32.0-2",
     "v4l2loopback-dkms": "0.15.4-2",
@@ -618,7 +618,7 @@
     "wayland-protocols": "1.49-1",
     "webrtc-audio-processing-1": "1.3-5",
     "which": "2.25-1",
-    "wireplumber": "0.5.17-1",
+    "wireplumber": "0.5.17-2",
     "wl-clipboard": "1:2.3.0-1",
     "woff2-font-awesome": "7.3.1-1",
     "wpa_supplicant": "2:2.12-1",
-- 
2.55.0

